service/src/test/kotlin/io/provenance/digitalcurrency/consortium/TestUtil.kt

```diff
@@ -0,0 +1,97 @@
+package io.provenance.digitalcurrency.consortium
```

Comment:
anything new here or just moving files?

Reply:
I renamed the file, added some default events, and cleaned up a dupe response function.